crates/hir/src/body.rs

```diff
@@ -955,7 +955,7 @@ mod tests {
                 let mutable_text = crate::testing::format_mutable(*mutable);
                 format!("{}let {mutable_text}{name} = {expr_str}\n", indent(nesting))
             }
-            Stmt::ExprStmt {
+            Stmt::Expr {
```

Comment:
**Issue**: The debug output function 'debug_stmt' still references the old 'ExprStmt' which could lead to runtime errors or incorrect debugging output if not updated. <br> **Fix**: Update the debug statements to reflect the new naming convention to ensure consistency and prevent potential runtime errors during debugging. <br> **Code Suggestion**: 
 ```
 -            Stmt::ExprStmt {
 +            Stmt::ExpressionStatement {
 ```